TU20Bot/Models/AccountModel.cs

```diff
@@ -0,0 +1,57 @@
+using System.Collections.Generic;
+using System;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Bson;
+
+namespace TU20Bot.Models {
+    public class AccountModel {
+        public const int HASH_SIZE = 24;
+        private const int ITERATIONS = 100000;
```

Comment:
I would even use something like [bcrypt](https://github.com/BcryptNet/bcrypt.net). It's a nice standard that moves our hashing code outside of this model. :D I also think its a bit safer (each hash generated is not the same & we don't need to store the hash in the model).

Reply:
Sounds good.